src/content/cloud/namespaces.mdx

```diff
@@ -31,4 +31,16 @@ To share a namespace go to the [Okteto Cloud UI](https://cloud.okteto.com/#/?ori
 
 <p align="center"><Image src="/docs/cloud/images/share.png" alt="share a namespace" width="850" /></p>
 
-In the `Share` dialog, type the email address of the team members you want to share this namespace with. Once you press save, Okteto Cloud will update the necessary permissions and notify your teammates via email.
\ No newline at end of file
+In the `Share` dialog, type the email address of the team members you want to share this namespace with. Once you press save, Okteto Cloud will update the necessary permissions and notify your teammates via email.
+
+One of the best way to reduce cost while working is to put your namespace on sleep. A week has 168 hours and let's assume
+your team works for 40 hours per week. Hence, about 128 hours per week, your team is not working. You can put your workloads
+to sleep for the time team is not working on it and wake the workloads when required.
+
+You can sleep your namespace from the command line with `okteto namespace sleep <name-of-namespace>`.
+
+When you want to wake your namespace, you can use the command `okteto namespace wake <name-of-namespace>`.
+
+You can sleep your namespace from Okteto Cloud itself by clicking on Sleep dialog.
+
+In order to wake the namespace, you can click on Wake dialog.
```

Comment:
```suggestion
In order to wake the namespace, you can click on Wake dialog.

:::tip

You can specify a global, default threshold for automatically sleeping a namespace in the [garbage collection](/self-hosted/administration/configuration/#gc) configuration.

:::

```

Reply:
@NitishKumar06 Please 😄

If you commit the suggestion I think we're set on this one 👍

Reply:
completely missed this one! Thanks for reminding me - Changes are applied!